### PR TITLE
Added an extra check on the log of the URL when open aq failures occur

### PR DESCRIPTION
**Pull request by rstrange-scottlogic**

### CODE REVIEW

File: air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py [hunk 1]
```diff
@@ -262,6 +262,7 @@ def test__in_situ_etl__timeouts_retry_twice_then_stop(
         main()
 
     assert "Response for London contained no results" in caplog.text
+    assert "URL was: https://api.openaq.org//v2/measurements?limit=3000" in caplog.text
```
Comment (mwalker-scottlogic) [inline air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py:265]: I think this has an extra /

File: air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py [hunk 2]
```diff
@@ -281,6 +282,7 @@ def test__in_situ_etl__internal_error_fails_without_retry(
...
```
Comment (mwalker-scottlogic) [inline air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py:285]: /

File: air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py [hunk 3]
```diff
@@ -304,6 +306,8 @@ def test__in_situ_etl__timeout_followed_by_success_returns_correctly(
         main()
 
     assert "Response for London contained no results" not in caplog.text
+    assert ("URL was: https://api.openaq.org//v2/measurements?limit=3000" not in
```
Comment (mwalker-scottlogic) [inline air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py:309]: /

**Review by mwalker-scottlogic — CHANGES_REQUESTED** [review]
Just a URL change needed I think

File: air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py (hunk 1 above)
Comment (rstrange-scottlogic) [reply air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py]: This was actually wrong in the code as well, have tweaked it there too

**Review by mwalker-scottlogic — APPROVED** [review]
lgtm

**Pull request merged**